site/components/pages/Infinity/cards.ts

```diff
@@ -0,0 +1,69 @@
+import olympusLogo from "public/logo-olympus-light.png";
+import polygonLogo from "public/logo-polygon-white.png";
+import markCubanLogo from "public/logo-mark-cuban.png";
+import stadiusLogo from "public/logo-stadius.png";
+
```

Comment:
Just for clarification: All external links int his file are not KlimaDao related and do not have to moved to constants IMO